apps/wallet-mobile/src/WalletInit/WalletCredentials/WalletCredentialsScreen.tsx

```diff
@@ -41,7 +44,7 @@ export const WalletCredentialsScreen = () => {
           isLoading || isSuccess
```

Comment:
This logic better outside? (nitpicking)

Reply:
this one is looking weird, won't touch this one, don't want to change the current behaviour, yet definitely worth a refactor.